crates/matrix-sdk-indexeddb/src/safe_encode.rs

```diff
@@ -234,11 +234,11 @@ impl<T: SafeEncode + ?Sized> SafeEncode for &T {
     }
 }
 
-impl<T: SafeEncode + ?Sized> SafeEncode for Box<T> {
+/* impl<T: SafeEncode + ?Sized> SafeEncode for Box<T> {
```

Comment:
What's up with this? Explain please in a comment or remove it.

Reply:
Oh, that was supposed to be removed 🙂